### ISSUE TITLE
Search tool on the toolbar has a bug under version 2.0.3 flash standalone version

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#330]
```
1.open a swf file,don't click anywhere
2.input some words when the doc is loaded completely
3.press the enter key

Then the flash player of debug version will throw an error, can't find the 
property or method, in the Viewer.searchText method.But the second time I do 
the same thing,it will be OK!Oh,by the way, the swf file in Example\flash\debug 
folder is OK.

I am using Windows 7 64bit, IE 8.


```

Original issue reported on code.google.com by `tzr...@gmail.com` on 4 Dec 2012 at 12:33

**Comment by GoogleCodeExporter** [comment]
```
are you using the Flex 4.6 version or the Flex 3.5 version?
```

Original comment by `erik.eng...@devaldi.com` on 19 Dec 2012 at 5:31

**Comment by GoogleCodeExporter** [comment]
```
I just using your swf file.The standalone version.
```

Original comment by `tzr...@gmail.com` on 22 Dec 2012 at 6:26

**Comment by GoogleCodeExporter** [comment]
Original comment by `erik.eng...@devaldi.com` on 27 Dec 2012 at 12:39
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed as of 2.0.6
```

Original comment by `erik.eng...@devaldi.com` on 24 Mar 2013 at 9:20
- Changed state: **Fixed**
